test/es-module/test-esm-repl-imports.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const path = require('path');
+const { spawn } = require('child_process');
+
+const child = spawn(process.execPath, [
+  '--interactive',
+], {
+  cwd: path.join(__dirname, '../fixtures/es-modules/pkgimports'),
```

Comment:
Please use `const fixtures = require('../common/fixtures')`
```suggestion
  cwd: fixtures.path('es-modules', 'pkgimports'),
```